EIPS/eip-4361.md

```diff
@@ -210,15 +253,13 @@ Additional functional requirements:
 5. Extensible
 
 ### Technical Decisions
-- Why EIP-191 (Signed Data Standard) over [EIP-712](./eip-712.md) (Ethereum typed structured data hashing and signing)
-    - EIP-191 is already broadly supported across wallets UX, while EIP-712 support for friendly user display is pending. **(1, 2, 3, 4)**
-        - https://github.com/LedgerHQ/app-ethereum/issues/105
-        - https://github.com/trezor/trezor-firmware/pull/1568
-    - EIP-191 is simple to implement using a pre-set prefix prior to signing, while EIP-712 is more complex to implement requiring the further implementations of a bespoke Solidity-inspired type system, RLP-based encoding format, and custom keccak-based hashing scheme. **(2)**
-    - EIP-191 produces more human-readable messages, while EIP-712 creates signing outputs for machine consumption, with most wallets not displaying the payload to be signed in a manner friendly to humans. **(1)**![](https://i.imgur.com/yvhz2mD.png)
-
-    - EIP-712 has the advantage of on-chain representation and on-chain verifiability, such as for their use in metatransactions, but this feature is not relevant for the specification's scope. **(2)**
-- Why not use JWTs? Wallets don't support JWTs. The keccak hash function is [not assigned by IANA](https://www.iana.org/assignments/jose/jose.xhtml) for use as a JOSE algorithm. **(2, 3)**
+- Why [EIP-191](./eip-191.md) (Signed Data Standard) over [EIP-712](./eip-712.md) (Ethereum typed structured data hashing and signing)
```

Comment:
I think EIP-712 is supported in all major wallets? I realize it isn't final status and we are working on it - I wouldn't want that to affect your decision on using 191 and not 712.

Reply:
This specifically dealt with display to the end-user. Specifically where the wallet has no custom view for something like SIWE and defaults to the basics - the person is presented with a wall of JSON [712] instead of a personal sign message which has the advantage of being human-readable.